analyzers/packaging/SonarAnalyzer.CSharp.nuspec

```diff
@@ -22,10 +22,10 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-    <file src="..\src\SonarAnalyzer.CSharp\bin\Release\net46\SonarAnalyzer.CFG.dll" target="analyzers" />
-    <file src="..\src\SonarAnalyzer.CSharp\bin\Release\net46\SonarAnalyzer.dll" target="analyzers" />
-    <file src="..\src\SonarAnalyzer.CSharp\bin\Release\net46\SonarAnalyzer.CSharp.dll" target="analyzers" />
-    <file src="..\src\SonarAnalyzer.CSharp\bin\Release\net46\Google.Protobuf.dll" target="analyzers" />
+    <file src="Binaries\Google.Protobuf.dll" target="analyzers" />
```

Comment:
Could you please tell me where is the Binaries folder coming from? I don't see where is it defined?

Reply:
There's a target in csproj that does this:

https://github.com/SonarSource/sonar-dotnet/blob/39f0d33a9e767b341cdea68a634c6cb2c6ee4d21/analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj#L51-L59

and

https://github.com/SonarSource/sonar-dotnet/blob/39f0d33a9e767b341cdea68a634c6cb2c6ee4d21/analyzers/src/SonarAnalyzer.VisualBasic/SonarAnalyzer.VisualBasic.csproj#L30-L37

Reply:
Both places were actually updated in this PR 😄 - C# was sorted, VB.NET was sorted and added CFG

Reply:
Yes, and now I found where the folder actually comes from, as it is just used in the `.csproj` files and not defined there.
https://github.com/SonarSource/sonar-dotnet/blob/39f0d33a9e767b341cdea68a634c6cb2c6ee4d21/analyzers/src/Directory.Build.targets#L7

So I think it would be nice to change the casing from Binaries to binaries, for the sake of consistency.

Reply:
Maybe a comment in the Directory.Build.targets mentioning that the path is also used here would ensure that the packaging does not fail if somebody in the future changes the value of the `BinariesFolder`.

Reply:
`Directory.Build.targets` is not the right place for such comment, but it makes sense to leave us some note.

It should also fail Java ITs after this PR. And I've created #5442 to add NuGet IT in the future.

Reply:
If somebody changes `BinariesFolder` in `Directory.Build.targets`, the build will actually fail hard.